dev-python/svg-path/svg-path-3.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="svg.path"
+
+DESCRIPTION="SVG path objects and parser"
+HOMEPAGE="https://github.com/regebro/svg.path"
+SRC_URI="https://github.com/regebro/svg.path/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+MODNAME="svg"
```

Comment:
Why do you need to define this var?

Reply:
You're right, it doesn't need to be in global scope, but it needs to be defined at install time, otherwise I get: http://bpaste.net/show/68d0f4f1ca37

Reply:
Well, obviously you need to stop using it once you remove it ...